content/index.md

```diff
@@ -0,0 +1,172 @@
+
+### Data Comands
+
+Any command that returns data should do so in the following ways.
+
+**Single value**
+
+A command that returns a single value can simply output that value.
+
+**Multiple values**
+
+Even for a single row, this data should be returned in columnar format, with headings. Example:
+
+```
+msisdn       | country | type       | features  | voiceCallbackType | voiceCallbackValue                   | moHttpUrl                                        | voiceStatusCallbackUrl
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+442038795229 | GB      | landline   | VOICE     | app               | 3050a697-f298-4601-8476-7805eafd33fc | undefined                                        | undefined             
+442038931433 | GB      | landline   | VOICE     | app               | 048f0fb1-39a8-40c4-1149-6a2260771fa1 | undefined                                        | undefined             
+442038931435 | GB      | landline   | VOICE     | app               | 048f0fb1-39a8-40c4-9d11-6a2260771fa1 | undefined                                        | undefined             
+447418340330 | GB      | mobile-lvn | VOICE,SMS | undefined         | undefined                            | undefined                                        | undefined             
+447418343340 | GB      | mobile-lvn | VOICE,SMS | undefined         | undefined                            | https://example.ngrok.io/webhooks/inbound-sms | undefined             
+447418343838 | GB      | mobile-lvn | VOICE,SMS | app               | a7fb2d02-92d6-43b4-ae58-6011eae58bbc | https://example.ngrok.io/incoming/sms         | undefined             
+```
+
+This standard approach lends itself well to use with common commands such as `cut` and `tail` that are in common use by developers.
+
+**JSON Format**
+
+For commands that return multiple values, adding a `--format-json` switch will return a pretty-printed JSON response instead of the standard data output.
+
+**Pagination**
+
+Especially for "power users" the size of the data responses can be large. All commands that return multiple values support common pagination parameters:
+* `--limit` The number of records to return. Default: 100
+* `--offset` How many records to skip before returning data. Default: 0
+
+## Error Handling
+
+When an error occurs, either in the CLI or in the underlying API response, the tool needs to respond in a predictable manner, including:
+
+* return an error exit status
+* show the detail of any error from the API
+* add any additional explanation, links or other references to explain to the user what went wrong or what they should do next
+
+## Delight Factor
+
+Tools are best when they assist the user to achieve their goals. The features in this section are here to enable developers to speedy success.
+
+### Autocomplete
```

Comment:
https://www.dropbox.com/s/9zjuylft4hiupft/Screenshot%202018-12-19%2011.15.54.png?dl=0

Typo suggestions would be awesome as well

Reply:
Added, love this suggestion